CMakeLists.txt

```diff
@@ -244,6 +244,15 @@ function(love_disable_warnings ARG_TARGET)
 	target_compile_options(${ARG_TARGET} PRIVATE ${DISABLE_WARNING_FLAG})
 endfunction()
 
+#
+# tracy
+#
+
+add_definitions(-DTRACY_ENABLE -DTRACY_IMPORTS -DTRACY_ON_DEMAND)

```

Comment:
Instead of doing this unconditionally, you could check with cmake if there is a specific file present inside the `src/libraries/tracy` folder, if you want people to allow skipping tracy.
Additionally, another idea would be to always include the clientside part of tracy in the libraries folder, but choosing between those two is up to the maintainers of this library.